boards/arm/96b_argonkey/96b_argonkey.dts

```diff
@@ -60,15 +60,24 @@
 		reg = <1>;
 		spi-max-frequency = <1000000>;
 		irq-gpios = <&gpiob 1 0>;
-		label = "LSM6DSL-SPI";
+		cs-gpios = <&gpioc 2 0>;
```

Comment:
cs-gpios should be up at the controller level.

This would be a good test of codegen.

Reply:
If cs-gpios is up at the controller level, it should then be populated in spi driver side.
This will work for sure on codegen side, but requires some extra work in spi API and lsm6dsl drivers.